docs/src/ExtendingCLIMA/Common/MoistThermodynamics.md

```diff
@@ -84,3 +90,132 @@ ts = PhaseNonEquil(param_set, e_int, ρ, PhasePartition(q_tot, q_liq, q_ice));
 T = air_temperature(ts);
 p = air_pressure(ts);
 ```
+
+## Extending
+
+If MoistThermodynamics.jl does not have a particular thermodynamic constructor that is needed, one can implement a new one in `CLIMA/src/Common/MoistThermodynamics/states.jl`. In this constructor, one must add whichever arguments they wish to offer as inputs, then translate this thermodynamic state into one of:
+
+ - `PhaseDry` a dry thermodynamic state, uniquely determined by two independent thermodynamic properties
+ - `PhaseEquil` a moist thermodynamic state in thermodynamic equilibrium, uniquely determined by three independent thermodynamic properties
+ - `PhaseNonEquil` a moist thermodynamic state in thermodynamic non-equilibrium, uniquely determined by four independent thermodynamic properties
+
+For example, to add a thermodynamic state constructor that accepts temperature, density and total specific humidity, we could add the following code to states:
+
+```
+"""
+    TemperatureSHumEquil_given_density(param_set, T, ρ, q_tot)
+
+Constructs a [`PhaseEquil`](@ref) thermodynamic state from temperature.
+
+ - `param_set` parameter set, used to dispatch planet parameter function calls
+ - `T` temperature
+ - `ρ` density
+ - `q_tot` total specific humidity
+"""
+function TemperatureSHumEquil(
+    param_set::APS,
+    T::FT,
+    ρ::FT,
+    q_tot::FT,
+) where {FT <: Real}
+    q = PhasePartition_equil(param_set, T, ρ, q_tot)
+    e_int = internal_energy(param_set, T, q)
+    return PhaseEquil{FT, typeof(param_set)}(param_set, e_int, ρ, q_tot, T)
+end
+```
+
+## Tested Profiles
+
+MoistThermodynamics.jl is tested using a set of profiles, or thermodynamic state regimes, specified in [`tested_profiles`](@ref MoistThermodynamics.tested_profiles).
+
+### Dry Phase
+
+```@example
+using CLIMA.MoistThermodynamics
+MT = MoistThermodynamics
+using CLIMAParameters
+using CLIMAParameters.Planet
+using Plots
+
+struct EarthParameterSet <: AbstractEarthParameterSet end;
+const param_set = EarthParameterSet();
+
+T_surface = 350
+T_min = 150
+FixedLapseRateFunc(FT) = FixedLapseRate{FT}(FT(T_surface), FT(T_min))
```

Comment:
This will be removed, so should be changed here.

Reply:
I included this for comparison with what we used to have (which we should be able to see in the docs)